audio.go

```diff
@@ -21,6 +28,7 @@ type AudioRequest struct {
 	Prompt      string // For translation, it should be in English
 	Temperature float32
 	Language    string // For translation, just do not use it. It seems "en" works, not confirmed...
+	Format      string
```

Comment:
Let's maybe make typechecker work for us a little bit here?

```go
type AudioResponseFormat string
const (
	AudioResponseFormatJSON AudioResponseFormat = "json"
	AudioResponseFormatSRT  AudioResponseFormat = "srt"
	AudioResponseFormatVTT  AudioResponseFormat = "vtt"
)

type AudioRequest struct {
...
	Format AudioResponseFormat
}
```

Reply:
Good idea, done